Changes

```diff
@@ -378,7 +378,11 @@ Next version (4.05.0):
   (Alain Frisch, report by Anil Madhavapeddy)
 
 - PR#7443, GPR#990: spurious unused open warning with local open in patterns
-  (Florian Angeletti, report by Gabriel Scherer)
+  (Florian Angeletti, report by Gabriel Scherer
+
+- GPR#406: remove polymorphic comparison for Types.constructor_tag in compiler
+  (Dwight Guth,
+   review by Gabriel Radanne, Pierre Chambart, Mark Shinwell)
```

Comment:
I think that multi-line credits are acceptable, and in particular may be used to respect the 80-columns rule. I'm not sure we care about which wrapping strategy is used.